src/components/typography/Body.tsx

```diff
@@ -16,6 +16,12 @@ export const Body = ({ size = 'default', children, style }: BodyProps) => {
             fontSize: 16,
             lineHeight: 19,
         };
+    } else if (size == 'semi-bold') {
+        bodyStyle = {
+            fontSize: 16,
+            lineHeight: 19,
+            fontWeight: '600',
```

Comment:
Should'nt we change the font familly to 'PublicSans-SemiBold' ?

Reply:
yes we should. Let me fix it